types/tslint.json

```diff
@@ -1,22 +1,7 @@
 {
```

Comment:
switching to eslint should be as easy as adding the typescript plugin and 4c rules and removing tslint, if you wanted